hypothesis_bio/hypothesis_bio.py

```diff
@@ -245,10 +245,10 @@ def fasta(
     if comment_source is None:
         comment_source = text(alphabet=characters(min_codepoint=32, max_codepoint=126))
     if sequence_source is None:
-        sequence_source = dna()
+        sequence_source = dna(min_size=1)
 
     comment = draw(comment_source)
-    sequence = draw(sequence_source)
+    sequence = draw(sequence_source)  #### TODO: has to be at least 1 char
```

Comment:
You could have a line below that ensures this 

`assume(len(sequence) > 0)` which will ensure it is always at least 1 char long.

Reply:
> I don't understand the purpose of the `to_filehandle.py` file and associated test file?

It's because, bio python and possibly other programs only take in file handler or path as input.
[Biopython SeqIO](https://biopython.org/wiki/SeqIO)

rn there are errors from importing **scikit**